c#/Mandoline.Api.Examples/Core/Client/ServiceModels/SelectionDto.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace Core.Client.ServiceModels;
+
+public enum FrequencyEnum
```

Comment:
Personally I do not like having Enums defined within a single file and I would normally adopt a policy of having a separate Enums folder with a file for each of the different Enums but that is just my personal preference.